docs/components/pagination.md

```diff
@@ -11,275 +11,298 @@
 
-<KPagination :totalCount="1000" :neighbors="2"/>
+<KPagination :total-count="1000" :neighbors="2" />
 
 ```html
-<KPagination :totalCount="1000" :pageSize="15" neighbors="2"/>
+<KPagination :total-count="1000" :neighbors="2" />
 ```
 
 ### disablePageJump
 
-Restrict navigation to only `previous` / `next` page. Defaults to `false`.
+Restrict navigation to only previous/next page. Defaults to `false`.
 
 <KComponent :data="{ letters: ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i'], visibleLetters: ['a', 'b', 'c']}" v-slot="{ data }">
   <div>
     <span><b>Visible letters: </b></span>
-    <span v-for="number in data.visibleLetters">{{ number }} </span>
-    <KPagination :items="data.letters" :totalCount="data.letters.length" :pageSizes="[3]" :disablePageJump="true" @pageChanged="({visibleItems}) => data.visibleLetters = visibleItems"/>
+    <span v-for="letter in data.visibleLetters">{{ letter }} </span>
+    <KPagination :items="data.letters" :total-count="data.letters.length" :page-sizes="[3]" disable-page-jump @page-change="({visibleItems}) => data.visibleLetters = visibleItems" />
   </div>
 </KComponent>
 
-```html
+```vue
 <template>
-  <div>
-    <span><b>Visible letters: </b></span>
-    <span v-for="number in data.visibleLetters">{{ number }} </span>
-    <KPagination
-      :items="data.letters"
-      :totalCount="data.letters.length"
-      :pageSizes="[3]"
-      :disablePageJump="true"
-      @pageChanged="({visibleItems}) => data.visibleLetters = visibleItems"/>
-  </div>
+  <span><b>Visible letters: </b></span>
+  <span v-for="letter in visibleLetters">{{ letter }}</span>
+  <KPagination
+    disable-page-jump
+    :items="letters"
+    :total-count="letters.length"
+    :page-sizes="[3]"
+    @page-change="({visibleItems}) => visibleLetters = visibleItems" />
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue'
-
-export default defineComponent({
-  setup() {
-    const letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i']
-    const visibleLetters = ['a', 'b', 'c']
-
-    return {
-      letters,
-      visibleLetters,
-    }
-  }
-})
+<script setup lang="ts">
+const letters = ref<string>(['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i'])
+const visibleLetters = ref<string>(['a', 'b', 'c'])
 </script>
 ```
 
 ### currentPage
 
-Manually control the current page instead of using native handling. If using this prop you MUST keep it up-to-date using the `@pageChanged` event in order to remain reactive to clicking the prev, next, and specific page buttons.
+Manually control the current page instead of using native handling. If using this prop you **must** keep it up-to-date using the `@page-change` event in order to remain reactive to clicking the prev, next, and specific page buttons.
```

Comment:
question: what is "native handling"? Description here should be a little more clear

Reply:
Updated